core/templates/dev/head/domain/classifier/AnswerClassificationResultObjectFactorySpec.ts

```diff
@@ -16,25 +16,67 @@
  * @fileoverview Unit tests for the AnswerClassificationResultObjectFactory.
  */
 
-require('domain/classifier/AnswerClassificationResultObjectFactory.ts');
+import { AnswerClassificationResultObjectFactory } from
+  'domain/classifier/AnswerClassificationResultObjectFactory.ts';
+
 require('domain/exploration/OutcomeObjectFactory.ts');
 require(
   'pages/exploration-player-page/services/answer-classification.service.ts');
 
-describe('Answer classification result object factory', function() {
-  var oof, acrof;
-  var DEFAULT_OUTCOME_CLASSIFICATION;
+class MockSubtitledHtml {
+  _html: any;
+  _contentId: any;
+  constructor(html, contentId) {
```

Comment:
I really like how you added the typeinfo in the previous file (e.g. ": string", etc.)

Why not do it here and elsewhere too?

Reply:
Done. Did some more typing. PTAL!